src/Writing/PostmanCollectionWriter.php

```diff
@@ -117,8 +117,8 @@ protected function resolveHeadersForRoute($route)
     protected function makeUrlData($route)
     {
         // URL Parameters are collected by the `UrlParameters` strategies, but only make sense if they're in the route
-        // definition. Filter out any URL parameters that don't appear in the URL, and assume they're query params.
-        [$urlParams, $queryParams] = collect($route['urlParameters'])->partition(function ($_, $key) use ($route) {
+        // definition. Filter out any URL parameters that don't appear in the URL.
+        $urlParams = collect($route['urlParameters'])->filter(function ($_, $key) use ($route) {
```

Comment:
Do you still need the filter then?

Reply:
Yeah. Postman collections need a matching `:param` reference in the URI for URL parameters. I can't remember what it did when there wasn't a matching pair - it's been a while since I was playing around with this stuff in Postman.